src/commands/PublishCommand.js

```diff
@@ -310,13 +310,8 @@ export default class PublishCommand extends Command {
   }
 
   gitCommitAndTagVersionForUpdates() {
-    let message = "Publish" + EOL;
```

Comment:
Shouldn't we only change the `message` part? The `tags` part should stay the same.

```bash
# commit message now
Publish

- package-1@1.0.1
- package-2@1.1.1

# commit message with custom message
chore: release :rocket:

- package-1@1.0.1
- package-2@1.1.1
```